### PR TITLE
[CP-2881] Windows Mudita Center is not closing when Mocking Mechanizm is used

### PR DESCRIPTION
**Pull request by lkowalczyk87**
JIRA Reference: [CP-2881]

### :memo: Description ️

-

### :muscle: Checklist before requesting a review - nice to have

- getState function in async thunk actions is correctly typed
- redux selectors are used in components / prop drilling is reduce

### :exclamation: Checklist before merging a pull request

- change went through the QA process
- translations are updated in dedicated application
- [CHANGELOG.md](./CHANGELOG.md) is updated

[CP-2881]: https://appnroll.atlassian.net/browse/CP-2881?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/mudita-center-e2e/src/consts/mockresponse-const.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export const mockResponseTomasz = {
```
Comment (dkarski) [inline apps/mudita-center-e2e/src/consts/mockresponse-const.ts:6]: Could we reword this variable to better describe its use case from a technical perspective? 🙏

File: apps/mudita-center/src/main.ts [hunk 2]
```diff
@@ -281,6 +281,11 @@ if (!gotTheLock) {
   // eslint-disable-next-line @typescript-eslint/no-misused-promises
   app.on("ready", createWindow)
 
+  app.on("before-quit", () => {
+    stopServer()
+  })
+  app.on("will-quit", () => {})
```
Comment (dkarski) [inline apps/mudita-center/src/main.ts:287]: Is the empty callback needed here? If so, why?

File: libs/e2e-mock/server/src/lib/server.tsx [hunk 3]
```diff
@@ -9,12 +9,17 @@ import {
   addKompaktValidator,
 } from "./mock-descriptor/mock-descriptor-validators"
 import { mockDescriptor } from "./mock-descriptor/mock-descriptor"
+import logger from "Core/__deprecated__/main/utils/logger"
 
 ipc.config.id = "MC"
-ipc.config.retry = 1500
+// ipc.config.retry = 1500
```
Comment (dkarski) [inline libs/e2e-mock/server/src/lib/server.tsx:15]: comment

**Review by dkarski — COMMENTED** [review]
I see a few logs have been added. Is that okay?

File: apps/mudita-center-e2e/src/consts/mockresponse-const.ts (hunk 1 above)
Comment (lkowalczyk87) [reply apps/mudita-center-e2e/src/consts/mockresponse-const.ts:6]: IMO it should be removed, I left it for testing purposes and I will removed before merge

**Review by dkarski — DISMISSED** [review]

File: apps/mudita-center-e2e/src/consts/mockresponse-const.ts (hunk 1 above)
Comment (tomaszmaleckimudita) [reply apps/mudita-center-e2e/src/consts/mockresponse-const.ts:6]: We will revert changes before merging

File: apps/mudita-center-e2e/src/specs/overview/e2e-mock-sample.e2e.ts [hunk 4]
```diff
@@ -1,28 +1,35 @@
...
+    })
+  })
+
+  after(() => {
+    E2EMockClient.stopServer()
+    E2EMockClient.disconnect()
   })
-  it("Connect device", async () => {
+
+  it("Test Tomasz #1 - addDevice", async () => {
```
Comment (MateuszMudita) [inline apps/mudita-center-e2e/src/specs/overview/e2e-mock-sample.e2e.ts:20]: I guess we have similar situation like in previous file? Will this test name be change?

Comment (lkowalczyk87) [reply apps/mudita-center-e2e/src/specs/overview/e2e-mock-sample.e2e.ts:20]: yup

**Review by MateuszMudita — DISMISSED** [review]

File: apps/mudita-center-e2e/src/specs/overview/e2e-mock-sample.e2e.ts (hunk 4 above)
Comment (tomaszmaleckimudita) [reply apps/mudita-center-e2e/src/specs/overview/e2e-mock-sample.e2e.ts]: Cleaned

File: apps/mudita-center-e2e/src/consts/mockresponse-const.ts (hunk 1 above)
Comment (tomaszmaleckimudita) [reply apps/mudita-center-e2e/src/consts/mockresponse-const.ts]: Cleaned

**Review by MateuszMudita — APPROVED** [review]

**Review by dkarski — APPROVED** [review]

**Review by mkurczewski — APPROVED** [review]

**Pull request merged**